package.json

```diff
@@ -126,15 +130,39 @@
                         ".",
                         "..\\fracas\\lib"
                     ],
-                    "markdownDescription": "A list of paths to search for Racket collections.",
+                    "markdownDescription": "Paths to search for Racket collections. Relative to __Project Dir__",
                     "order": 3
                 },
                 "vscode-fracas.general.ninjaPath": {
                     "type": "string",
                     "default": "ninja",
-                    "markdownDescription": "Specifies the path to the `ninja` executable used to precompile racket files.",
+                    "markdownDescription": "Path to the `ninja` executable used to precompile racket files.",
                     "order": 4
                 },
+                "vscode-fracas.localization.stringTableRegistryFile": {
+                    "type": "string",
+                    "default": "..\\tdp1.unreal\\Source\\Data\\TdpLocalization.cpp",
+                    "markdownDescription": "Path to the generated C++ source file that registers localized text tables with the UE4 project. Relative to __Project Dir__",
+                    "order": 5
+                },
+                "vscode-fracas.localization.stringTableSourcePaths": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "default": [
+                        "fracas\\localization-defines.frc",
```

Comment:
hhhrmm how do we socialize that this is separately maintained list from `project.rktd`?